webpack/scenes/ContentViews/Publish/CVPublishForm.js

```diff
@@ -61,6 +65,24 @@ const CVPublishForm = ({
                   <TextContent>{__('Newly published version will be the same as the previous version.')}</TextContent>
                 </Alert>)
             }
+            {!duplicateReposAlertDismissed && composite &&
+                (duplicateRepos !== null || duplicateRepos.length > 0) &&
+                (
+                <Alert
+                  ouiaId="duplicate-repos-alert"
+                  variant="info"
+                  isInline
+                  title={__('Duplicate repositories in component content views')}
+                  actionClose={
+                    <AlertActionCloseButton
+                      onClose={() => setDuplicateReposAlertDismissed(true)}
+                    />
+                        }
+                  style={{ marginBottom: '24px' }}
+                >
+                  <TextContent>{__('Duplicate repositories between the component content view versions will merge, resulting in a composite content view with all the content that exists among the duplicates.')}</TextContent>
```

Comment:
```suggestion
                  <TextContent>{__('Repositories in common between the content view versions will merge, resulting in a composite content view with all the content that exists among the duplicates.')}</TextContent>
```

Reply:
What do we mean by "all the content that exists among the duplicates?" Does this mean that if one CV version has something filtered out in a repository, but the other doesn't, I'll get the unfiltered content?

Reply:
Ya..So a union of content in the 2 repository versions between the 2 CVs..

Reply:
The message comes from the old UI which was here: https://github.com/Katello/katello/pull/8198/files

Reply:
Hmm.. How about
```
The composite content view will contain a combination of all the repositories from each of the content view versions.
```

Reply:
The composite content view will contain a combination of _all the content_ from the repositories from each of the content view versions.

Reply:
We decided on
```
Repositories common to the selected content view versions will merge, resulting in a composite content view that is a union of all content from each of the content view versions.
``

Reply:
Updated to the freshly baked text..